### PR TITLE
Added bunch of api

### PR DESCRIPTION
**Pull request by jzarca01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,10 +109,16 @@ Interested in adding your own API library, submit a pull request. Instructions c
 *API* | *Description* | *Language* | *Stars* | *Last Commit*
 --- | --- | --- | --- | ---
 [NHL API Documentation](https://gitlab.com/dword4/nhlapi) | Documenting the publicly accessible portions of the NHL API | English | [![badge](https://img.shields.io/badge/dynamic/json?url=https://gitlab.com/api/v4/projects/6582969&label=Stars&query=star_count&color=important&style=social)](https://gitlab.com/dword4/nhlapi) | [![badge](https://img.shields.io/badge/dynamic/json?url=https://gitlab.com/api/v4/projects/6582969&label=Last%20Activity&query=last_activity_at&color=important)](https://gitlab.com/dword4/nhlapi)
+[NobodySurf](https://github.com/jzarca01/node-nobodysurf) | An API for the video app NobodySurf | JavaScript | [![badge](https://img.shields.io/github/stars/jzarca01/node-nobodysurf.svg?style=social&label=Star&maxAge=2592000)](https://github.com/jzarca01/node-nobodysurf/stargazers) | [![badge](https://img.shields.io/github/last-commit/jzarca01/node-nobodysurf.svg)](https://github.com/jzarca01/node-nobodysurf)
```
Comment (Rolstenhouse) [inline README.md:115]: This doesn't look like the shields are resolving. Can you look into this?

**Review by Rolstenhouse — CHANGES_REQUESTED** [review]
Do you mind alphabetizing and ordering the repos? Thanks! Cool to see how many unofficial APIs you've added.

**Comment by jzarca01** [comment]
Done ✅

**Comment by Rolstenhouse** [comment]
Thanks!

**Pull request merged**